# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/hdparm/hotfixes.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./hdparm.c.orig	2003-02-10 12:58:22.000000000 +0100
+++ ./hdparm.c	2003-02-10 12:58:55.000000000 +0100
@@ -728,7 +728,9 @@
 	case (MD_MAJOR):
 #endif
 	case (VXVM_MAJOR):
+#ifdef LVM_BLK_MAJOR
 	case (LVM_BLK_MAJOR):
+#endif
 		is_scsi_hd = 1;
 		break;
 #ifdef XT_DISK_MAJOR
